### ISSUE TITLE
Conditional jump or move depends on uninitialised value(s) in PageItem::setLineEnd on importing from PDF.

### ISSUE THREAD

**Issue by Cyp** [scribusproject/scribus#119]
```valgrind
==29299== Conditional jump or move depends on uninitialised value(s)
==29299==    at 0x6AC802: PageItem::setLineEnd(Qt::PenCapStyle) (pageitem.cpp:4082)
==29299==    by 0x3EC2E7E6: SlaOutputDev::drawChar(GfxState*, double, double, double, double, double, double, unsigned int, int, unsigned int*, int) (slaoutput.cpp:3402)
==29299==    by 0x3F31EEDB: Gfx::doShowText(GooString const*) (Gfx.cc:4041)
==29299==    by 0x3F31F302: Gfx::opShowText(Object*, int) (Gfx.cc:3771)
==29299==    by 0x3F3183C7: Gfx::go(bool) (Gfx.cc:752)
==29299==    by 0x3F318A8E: Gfx::display(Object*, bool) (Gfx.cc:714)
==29299==    by 0x3F371BBD: Page::displaySlice(OutputDev*, double, double, int, bool, bool, int, int, int, int, bool, bool (*)(void*), void*, bool (*)(Annot*, void*), void*, bool) (Page.cc:548)
==29299==    by 0x3F371E87: Page::display(OutputDev*, double, double, int, bool, bool, bool, bool (*)(void*), void*, bool (*)(Annot*, void*), void*, bool) (Page.cc:469)
==29299==    by 0x3EC195F3: PdfPlug::convert(QString const&) (importpdf.cpp:771)
==29299==    by 0x3EC1B143: PdfPlug::import(QString const&, TransactionSettings const&, int, bool) (importpdf.cpp:271)
==29299==    by 0x3EC24B11: ImportPdfPlugin::import(QString, int) (importpdfplugin.cpp:226)
==29299==    by 0x3EC225FE: ImportPdfPlugin::loadFile(QString const&, FileFormat const&, int, int) (importpdfplugin.cpp:150)
```

**Comment by aoloe** [comment]
submitted to the official bug tracker as https://bugs.scribus.net/view.php?id=15700

**Comment by aoloe** [comment]
fixed through 3dbd84b9
